doc/releases/changelog-0.36.0.md

```diff
@@ -11,53 +11,80 @@
   [(#5154)](https://github.com/PennyLaneAI/pennylane/pull/5154)
   [(#5464)](https://github.com/PennyLaneAI/pennylane/pull/5464)
   [(#5465)](https://github.com/PennyLaneAI/pennylane/pull/5465)
+  [(#5278)](https://github.com/PennyLaneAI/pennylane/pull/5278)
+  [(#5384)](https://github.com/PennyLaneAI/pennylane/pull/5384)
 
-  Focusing on the [spectral norm](https://en.wikipedia.org/wiki/Matrix_norm)
-  error, it is possible to specify errors in the following operations:
+  Two new user-facing classes are now available to allow for calculating and propagating 
+  gate errors in PennyLane:
 
-  * For algorithmic-level usecases it is useful to create a custom operation representing a
-    major building block of the algorithm. This building block can have an error attached but
-    does not require a decomposition or matrix representation:
+  * `qml.resource.error.SpectralNormError`: the spectral norm error is defined as the 
+    distance, in [spectral norm](https://en.wikipedia.org/wiki/Matrix_norm), between 
+    the true unitary we intend to apply and the approximate unitary that is actually applied.
 
-    ```python
-    import pennylane as qml
-    from pennylane.resource.error import ErrorOperation, SpectralNormError
+  * `qml.resource.error.ErrorOperation`: a base class that inherits from `qml.operation.Operation` 
+    and represents quantum operations which carry some form of algorithmic error.
+
+  `SpectralNormError` can be used for back-of-the-napkin type calculations like obtaining the
+  spectral norm error between two unitaries via `get_error`:
 
+  ```python
+  import pennylane as qml
+  from pennylane.resource.error import ErrorOperation, SpectralNormError
+
+  intended_op = qml.RY(0.40, 0)
+  actual_op = qml.RY(0.41, 0) # angle of rotation is slightly off
+  ```
+  
+  ```pycon
+  >>> SpectralNormError.get_error(intended_op, actual_op)
+  0.004999994791668309
+  ```
+  
+  `SpectralNormError` is also the focus of specifying errors in the following ways:
+
+  * For operations representing a major building block of an algorithm, we can create a class
```

Comment:
Maybe something like:
```suggestion
  * To represent error-prone parts of an algorithm, we can create a class
```
Error-prone sounds more like noise (not `AlgorithmicError`) than error coming from an approximation (`AlgorithmicError`) though.